### DISCUSSION TITLE
SBOM file extension

### DISCUSSION THREAD

**Discussion by lucacome** [goreleaser/goreleaser#3457] [Ideas, Issue Triage, and General discussions]
Hello, we started using the SBOM generation option and I noticed the `.sbom` file extension. 
I couldn't find anything on the web for this extension so I looked at the SPDX spec and saw that they suggest using `.spdx.json` as a file extension when generating a JSON SBOM, so I was wondering if this should be the default extension and not `.sbom` or what was the reasoning behind this.

Thanks!

**Comment by caarlos0** [comment]
I'm not sure what the reasoning was, but it's unlikely we'll change it unless its really needed... users can use the extension they want in the config...

**Reply by lucacome** [reply]
Yeah you're right it can always be customized, but I personally try to use the default as much as possible...and it might be a little bit confusing that the default format is `spdx-json` but with an extension that doesn't match it.

Also I guess I was just curious where `.sbom` came from and if my google-fu was just weak because I couldn't find anything with that extension 😅

**Reply by caarlos0** [reply]
I honestly don't know either... it was added in this pr https://github.com/goreleaser/goreleaser/pull/2648

maybe @wagoodman can weight in?
